Maintenance and fixes (#1)
jwkoelewijn/consul-clojure: README.md

[Consul](https://www.consul.io) is an awesome service discovery and configuration provider.

```clojure
[consul-clojure "0.7.4"]
```

## Changelog

### 0.7.4

> Fixed `catalog-services` query to provide a hashmap wher the keys are the service name.

Reviewer: ```suggestion
Fixed `catalog-services` query to provide a map indexed by service name.
```

also fixes a typo :P

> Fixed `catalog-services` query to provide a hashmap wher the keys are the service name.

Reviewer 2: ```suggestion
Fixed `catalog-services` query to return a hashmap with service names as keys .
```
